web/src/pages/Home/Form/ContactForm.tsx

```diff
@@ -20,8 +21,23 @@ function ContactForm() {
         setFormData({ ...formData, [e.target.name]: e.target.value });
     };
 
-    const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    const handleSubmit = (e: any) => {
         e.preventDefault();
+        emailjs
+            .sendForm(
+                import.meta.env.VITE_REACT_APP_SERVICE_ID_CONTACT_FORM,
```

Comment:
can you also update docker-compose with default configuration, or add template values in the example `.env` in the `web/` directory